tiled/src/test/kotlin/ktx/tiled/TiledMapRendererTest.kt

```diff
@@ -0,0 +1,75 @@
+package ktx.tiled
+
+import com.badlogic.gdx.graphics.OrthographicCamera
+import com.badlogic.gdx.maps.tiled.renderers.BatchTiledMapRenderer
+import com.badlogic.gdx.maps.tiled.renderers.beginInternal
+import com.badlogic.gdx.maps.tiled.renderers.endInternal
+import com.badlogic.gdx.math.Matrix4
+import com.badlogic.gdx.math.Rectangle
+import io.kotlintest.mock.mock
+import org.junit.Assert.assertSame
+import org.junit.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+
+class TiledMapRendererTest {
+
+  @Test
+  fun `should call beginRender and endRender without setView`() {
+    val renderer = mock<BatchTiledMapRenderer>()
+
+    renderer.use {
+      verify(renderer).beginInternal()
+      assertSame(renderer, it)
+      verify(renderer, never()).endInternal()
+    }
+    verify(renderer).endInternal()
```

Comment:
Not sure if you can verify calls on essentially static methods, you might have to change the tests package as well to verify the protected methods.

Reply:
Why is that necessary? Isn't it sufficient to verify if the wrapper functions got called? Because the actual functions cannot be verified since they are protected. At least to my knowledge it is not advisable to verify protected/private methods.

Reply:
I thought it might have been a limitation of Mockito, but apparently not. The tests pass, so I'm fine with verifying the extension methods.